pages/docs/reference/whatsnew14.md

```diff
@@ -612,3 +612,34 @@ Complete the initial configuration, and when you add a new dependency to `cocoap
 The new dependency will be added automatically. No additional steps are required.
 
 Learn [how to add dependencies](native/cocoapods.html).
+
+## Scripting and REPL
+
+### New dependencies resolution API
+
+In 1.4, we present the new API for dependency resolution and its implementations. It is published in new artifacts
+`kotlin-scripting-dependencies` and `kotlin-scripting-dependencies-maven`. The previous dependencies resolution functionality
+in `kotlin-script-util` library is now deprecated.
+
+### New REPL API
+
+The new experimental REPL API is now a part of the scripting API. There are also several implementations of it in the
+published artifacts; some with advanced functionality, such as code completion. We use it in the Kotlin Jupyter kernel [uses this functionality](https://blog.jetbrains.com/kotlin/2020/05/kotlin-kernel-for-jupyter-notebook-v0-8/)
```

Comment:
"We use it in the Kotlin Jupyter kernel [uses this functionality]" - seems like a half-baked sentence to me.